flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/runtime/batch/sql/FunctionITCase.java

```diff
@@ -125,6 +128,27 @@ public void testUserDefinedTemporarySystemFunctionByUsingJar() throws Exception
         testUserDefinedFunctionByUsingJar(functionDDL, dropFunctionDDL);
     }
 
+    @Test
+    public void testCreateTemporarySystemFunctionWithTableAPI() {
```

Comment:
According to the related discussion in https://github.com/apache/flink/pull/20635, please delete these tests in batch mode.

Reply:
Make sense. I removed table api related test cases.